qiita_db/metadata_template/test/test_prep_template.py

```diff
@@ -1652,6 +1652,15 @@ def test_delete_samples(self):
             self.metadata, self.test_study, self.data_type)
         ptB = qdb.metadata_template.prep_template.PrepTemplate.create(
             self.metadata, self.test_study, self.data_type)
+
+        # first let's test that we cannot delete all samples from one of the
+        # preps
+        with self.assertRaises(ValueError) as e:
```

Comment:
You can use assertRaisesRegex to validate the error message and exception simultaneously.

https://docs.python.org/3.7/library/unittest.html#unittest.TestCase.assertRaisesRegex